src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNames.cs

```diff
@@ -393,9 +393,9 @@ internal static string MakeSynthesizedLocalName(SynthesizedLocalKind kind, ref i
             return null;
         }
 
-        internal static string MakeSynthesizedInstrumentationPayloadFieldName(MethodSymbol method, int methodOrdinal)
+        internal static string MakeSynthesizedInstrumentationPayloadLocalName(int uniqueId)
```

Comment:
In the prior incarnation, the name was actually for a field. Another case that assigns a name to a synthesized local for use in a display class uses the term "LocalField"--I'll change this to follow that pattern.